dags/iop/iop_process_file.py

```diff
@@ -1,12 +1,63 @@
+import base64
+import xml.etree.ElementTree as ET
+
 import airflow
+import requests
 from airflow.decorators import dag, task
+from common.enhancer import Enhancer
+from common.enricher import Enricher
+from iop.parser import IOPParser
+from jsonschema import validate
+
+
+def iop_parse_file(**kwargs):
+    if "params" in kwargs and "file" in kwargs["params"]:
+        encoded_xml = kwargs["params"]["file"]
+        xml_bytes = base64.b64decode(encoded_xml)
+        xml = ET.fromstring(xml_bytes.decode("utf-8"))
+
+        parser = IOPParser()
+        parsed = parser.parse(xml)
+
+        return parsed
+    raise Exception("There was no 'file' parameter. Exiting run.")
```

Comment:
maybe you can revert the condition (`if "params" in kwargs and "file" in kwargs["params"]`) and raise it at the beginning of the function?